docs/compatibility.md

```diff
@@ -108,6 +108,7 @@ can take a while as it will download and install the missing dependencies.
 A small time tax to pay for the ability to use any type of image.
 This will **not** occur after the first time, **subsequent enters will be much faster.**
 
+NixOS is not a supported container distro, and there are currently no plans to bing support to it. If you are looking for unprivlaged NixOS environments, we suggest you look into [nix-shell](https://nixos.org/manual/nix/unstable/command-ref/nix-shell.html). 
```

Comment:
Little typo *to bring

Reply:
I am horrible at typos. I seriously need a new keyboard.

Reply:
Don't worry, half of the whole project is a typo itself.